app/views/divisions/division_not_found.html.haml

```diff
@@ -0,0 +1,6 @@
+- content_for :title, 'Division not found'
+%h1= yield :title
+
+No Division found. If you entered a date, please make
```

Comment:
Sure. I can create something like it. It won't be very helpful for this scenario, but as you said, it is an rare error anyway. Any ideas on the wording?